src/meta/processors/BaseProcessor.inl

```diff
@@ -155,7 +155,7 @@ ErrorOr<nebula::cpp2::ErrorCode, std::vector<HostAddr>> BaseProcessor<RESP>::all
 template<typename RESP>
 ErrorOr<nebula::cpp2::ErrorCode, int32_t> BaseProcessor<RESP>::autoIncrementId() {
     folly::SharedMutex::WriteHolder holder(LockUtils::idLock());
-    static const std::string kIdKey = "__id__";
+    static const std::string kIdKey = MetaServiceUtils::idKey();
```

Comment:
What if that static map is not initialized at this time ? I mean I've heard that C++ not guarantee the sequence of two static.

Reply:
great catch. it looks like a [problem](https://isocpp.org/wiki/faq/ctors#static-init-order). removed `static` here. I think the RVO would take place.